Stellar-operation-manage-poll.x

```diff
@@ -89,7 +89,13 @@ enum ManagePollResultCode
     //: Not allowed to close poll which
     POLL_NOT_READY = -2,
     //: Only result provider is allowed to close poll
-    NOT_AUTHORIZED_TO_CLOSE_POLL = -3
+    NOT_AUTHORIZED_TO_CLOSE_POLL = -3,
+    //: End time is in the past
+    INVALID_END_TIME = -4,
+    //:Only poll owner and admin can cancel poll
+    NOT_AUTHORIZED_TO_CANCEL_POLL = -5,
+    //:Only poll owner can update end time
```

Comment:
or admin

Reply:
use one error code for both cases (not auth for cancel and update) - just `NOT_AUTHORIZED`
client knows which action he tries to apply